authority/provisioner/claims.go

```diff
@@ -110,6 +115,15 @@ func (c *Claimer) IsDisableRenewal() bool {
 	return *c.claims.DisableRenewal
 }
 
+// IsDisableSmallstepExtensions returns if the Smallstep extensions, like the
+// provisioner extension, should be excluded from the certificate.
+func (c *Claimer) IsDisableSmallstepExtensions() bool {
+	if c.claims == nil || c.claims.DisableSmallstepExtensions == nil {
+		return *c.global.DisableSmallstepExtensions
+	}
+	return *c.claims.DisableSmallstepExtensions
+}
+
```

Comment:
```suggestion
// IsDisableSmallstepExtensions returns whether Smallstep extensions, such as
// the provisioner extension, should be excluded from the certificate.
func (c *Claimer) IsDisableSmallstepExtensions() bool {
	if c.claims == nil || c.claims.DisableSmallstepExtensions == nil {
		return *c.global.DisableSmallstepExtensions
	}
	return *c.claims.DisableSmallstepExtensions
}

```
Generally I prefer `ShouldX` instead of `IsX` for "active function names". `IsX` would be better if it were `AreSmallstepExtensionsDisabled`.

Reply:
Fixed with https://github.com/smallstep/certificates/commit/3790bd01a496452bf1e13b534357fd9f215f2840